README.md

```diff
@@ -16,6 +16,7 @@ in the procedural macro API.
 ```rust
 const VERT: &[u32] = include_glsl!("shaders/example.vert");
 const FRAG: &[u32] = include_glsl!("shaders/example.glsl", kind: frag);
+const RGEN: &[u32] = include_glsl!("shaders/example.rgen", target: vulkan1_2); // vulkan1_2 is required to build with GL_EXT_ray_tracing
```

Comment:
Would it make sense to bump the default target to 1.2 automatically for raytracing-related shader types?